### ISSUE TITLE
responseXML access error

### ISSUE THREAD

**Issue by bpatrik** [jpillora/xhook#89]
Hi,

I got the following error in the latest version (`1.4.8`) in chrome `66.0.3359.181`

```
Uncaught DOMException: Failed to read the 'responseXML' property from 'XMLHttpRequest':
The value is only accessible if the object's 'responseType' is '' or 'document' (was 'text').
```
As far as I could debug, the error seems to be in the `readBody` function at `xhook.js:336`

Might be related to #86 

Any suggestions?
Thank you!
Patrik

**Comment by morsdyce** [comment]
Hi,

Thanks for reporting this, is there a sample request you can share so I can test it?
I currently do not see any issues from the tests for regular requests and XML requests

**Comment by bpatrik** [comment]
Hi,

I have something like this:

```typescrpit
const url = '...';
const request:= {
      url: 'https://raw.githubusercontent.com/bpatrik/PiGallery2/master/README.md',
      method: 'GET',
      timeout: 0,
      headers: {},
      type: 'text',
      withCredentials: false
    };

const xhr = new XMLHttpRequest();
xhr.open('GET', url + '/api/http-request?data=' + encodeURIComponent(JSON.stringify(req)), true);
if (request.type) {
     xhr.responseType = request.type;
}

xhr.send();
```

should I not set the `type` explicitly to `text` ?

**Comment by morsdyce** [comment]
Generally it's not something you do but rather define these things via
headers.

Either way I'll have a look if this type of request is compliant with the
spec and if it is I'll work on supporting it.

**Comment by morsdyce** [comment]
Fixed in v1.4.9

Please check it out and let me know if it works for you.

Thanks

**Comment by bpatrik** [comment]
Seems to be working.
Thank you!
